src/openms/source/FORMAT/HANDLERS/MzXMLHandler.cpp

```diff
@@ -1012,7 +1073,7 @@ namespace OpenMS
         s_completiontime_ = xercesc::XMLString::transcode("completionTime");
         s_precision_ = xercesc::XMLString::transcode("precision");
         s_byteorder_ = xercesc::XMLString::transcode("byteOrder");
-        s_pairorder_ = xercesc::XMLString::transcode("pairOrder");
+        s_contentType_ = xercesc::XMLString::transcode("contentType");
```

Comment:
this attribute is utterly useless in all mzXML schemata, since only one value is allowed anyway.
So every good parser should just ignore it -- or throw an error if it does not show the expected value (this is what OpenMS does).